File: sws_backend/SWS/src/main/java/it/unisa/emad/comunesalerno/sws/entity/StatoOperazione.java
package it.unisa.emad.comunesalerno.sws.entity;

public enum StatoOperazione {
    DA_APPROVARE("Da approvare"),APPROVATO("Approvato"),IN_MODIFICA("In modifica"),ANNULLATO("Annullato"),DA_CANCELLARE("Da cancellare");
    private String text;

    private StatoOperazione(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }
}

File: sws_frontend/lib/old_entity/StatoOperazione.dart
import 'package:flutter/material.dart';

class StatoOperazione extends StatelessWidget {
  static const String DA_APPROVARE = "Da approvare";
  static const String APPROVATO = "Approvato";
  static const String IN_MODIFICA = "In modifica";
  static const String ANNULLATO = "Annullato";

  final String text;
  const StatoOperazione({Key? key, required this.text}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Container();
  }
}
